lib/consultant/configuration.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Consultant
+  class Configuration
+    attr_accessor :results_directory, :timezone, :root
+
+    REQUIRED_SETTINGS = [
+      :results_directory,
+      :timezone,
+      :root
+    ]
+
+    def initialize
+      @results_directory = nil
+      @timezone = Time.zone
+      @root = File.expand_path '../..', __FILE__
+    end
+
+    def configured?
+      REQUIRED_SETTINGS.all? do |accessor|
```

Comment:
at some point, I would like to find a way to use this helper method to raise an error if the gem is not configured